README.md

```diff
@@ -94,8 +175,10 @@ The `tidb-insight` project is designed to intergrate with [tidb-ansible](https:/
 
 > All collecting features will be disabled by default after the development of intergration is finished, so it's highly recommended to specify all the arguments needed when using `tidb-insight` now, even if the feature is currently enabled by default.
 
-TO DO: Add documents/links to documents of using `tidb-insight` with `tidb-ansible`.
+The `tidb-insight` tarball is included in `tidb-ansible` as part of playbook `collect_diagnosis`, it deploies the pre-build `tidb-insight` dependencies and tools, as well as scripts to target server. Several tasks can be called directly from `ansible-playbook` such as collecting log files of all deployed TiDB/TiKV/PD instances (While `--log-auto` of `tidb-insight` only collects log files of running processes), and copy them back to the controller server in `.tar.gz` format.
```

Comment:
The `tidb-insight` tarball is included in `tidb-ansible` as part of playbook `collect_diagnosis`, it **deploys** the pre-build `tidb-insight` dependencies and tools, as well as scripts to **the** target server. Several tasks can be called directly from `ansible-playbook` such as collecting log files of all deployed TiDB/TiKV/PD instances (While `--log-auto` of `tidb-insight` only collects log files of running processes), and copy them back to the controller server in `.tar.gz` format.